CHANGELOG

```diff
@@ -1,6 +1,10 @@
 devel
 -----
 
+* Implement license changes: The community edition without a license now has a
```

Comment:
There are effectively no longer Community and Enterprise Editions from 3.12.5 onward, only an unlicensed or licensed Enterprise Edition if you will (plus you may compile openly available source code yourself, but that is different to the Community Edition, which has different license terms than just the source code). Maybe state that there is now only one release with all features of Enterprise Edition instead of saying something about the Community Edition here?
```suggestion
* Implement license changes: The Community Edition without a license now has a
```